modules/managing-feature-flags.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * configuration/managing-preview-features.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="managing-feature-flags_{context}"]
+= Managing Feature Flags
+
+== Prerequisites
+* Access to the environment where the {product-title-short} component is deployed.
+* Permission to modify environment variables.
+* Understanding the differences between `dev-preview` and `tech-preview` release stages. For more information see the link:https://access.redhat.com/articles/6966848[Developer and Technology Previews: How they compare].
+* Understanding if the feature flag needs to be configured before the deployment: see if the installation manifests make use of the required flag.
+
+.Procedure
+
+. Identify the Feature Flag.
+Consult the release notes or `/v1/featureflags` API endpoint to identify the flag for the feature you want to enable or disable.
+
+. Modify the Feature Flag.
+* To *enable* a feature, configure the deployment of the {product-title-short} component to include the environment variable, associated to the flag, with the value set to `true`.
+* To *disable* a feature, set the environment variable to `false`.
+
+. Verify the Changes.
+Once the application is restarted or redeployed, verify that the feature has been enabled or disabled as intended:
+
+* Check the output of the `/v1/featureflags` API endpoint.
+* Check the application functionality related to the feature.
+* Review logs or monitoring tools for any errors or confirmation messages.
+
+== Best Practices
+* *Dev-Preview*: Do not enable dev-preview features.
```

Comment:
- Lists need an introduction (same for troubleshooting) such as "Follow these best practices for using the feature flag":
- Normally we wouldn't use this format (bold item, then sentence) for best practices. If you are intent on using it, maybe use the definition list format? https://www.ibm.com/docs/en/ibm-style?topic=format-lists